Source/Core/Common/ArmFPURoundMode.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/CommonTypes.h"
+#include "Common/FPURoundMode.h"
+
+#ifdef _MSC_VER
+extern "C" {
+// Implemented in ArmFPURoundModeAsm.asm because MSVC doesn't support inline assembly for AArch64
+// and doesn't have any intrinsics for setting/getting FPCR
```

Comment:
you can use 
```c
_ReadStatusReg(ARM64_FPCR);
_WriteStatusReg(ARM64_FPCR, value);
```
to avoid assembly files here.

Reply:
Ah, so there is an intrinsic for it and I just didn't find it! Thank you, that makes it a lot simpler. (Probably half the time spent coding was spent just getting all the MSVC assembly stuff to work :( )